test_fms/parser/test_yaml_parser.sh

```diff
@@ -26,7 +26,7 @@
 . ../test-lib.sh
 
 if [ ! -z $parser_skip ]; then
-  SKIP_TESTS='test_yaml_parser.[1-22]'
+  SKIP_TESTS='test_yaml_parser.[1-21]'
```

Comment:
Is the change from 22 to 21 expected?

Reply:
There was a test to check if the parser would fail with a non-existent yaml file... but with the changes it shouldn't fail. I guess we could modify the test to check for the desired behavior (file_id = 999) instead of just removing it.

Reply:
Have been assured this is good